src/game_state.rs

```diff
@@ -90,6 +94,7 @@ impl event::EventHandler for GameState {
         Enemies::tick(self);
         Towers::tick(self);
         Projectiles::tick(self);
+        self.effects.tick();
```

Comment:
This is weird that all are called via `X::tick(self)` and this is `self.effects.tick()`

Reply:
thats because effects are not supposed to depend on the state, they are supposed to be visual only and therefore don't get access to self, only to self.effects.